differ/differ.go

```diff
@@ -438,14 +439,19 @@ func generateWeeklyNotificationMessage(advisorURI string, accountID string, dige
 	return
 }
 
-// appendEventToNotificationMessage function adds a new event to the given notification message after constructing the payload string
-func appendEventToNotificationMessage(ruleURI string, notification *types.NotificationMessage, ruleName string, totalRisk int, publishDate string, clusterID string, module string, errorKey string) {
+func generateNotificationPayloadURL(ruleURI string, clusterID string, module string, errorKey string) (notificationPayloadURL string) {
```

Comment:
just for the future - it is error prone to have "typeless" values. In the future we need to improve it to use proper types (and possibly structures instead of bunch of parameters).